com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/FileSystemConfiguration.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+package com.ca.lsp.cobol;
+
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.lsp4j.WorkspaceFolder;
+
+import javax.annotation.Nonnull;
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+public class FileSystemConfiguration extends ConfigurableTest {
```

Comment:
FileSystemConfigurationTest

Reply:
The purpose of this class is to act as a provider for methods reused in other unit tests.
Have nothing to test so I marked intentionally without ''Test'' in the name.
I will add a comment next to the class to clarify that